extras/container/heketi-start.sh

```diff
@@ -21,6 +27,60 @@ fail() {
     exit 1
 }
 
+archive_db() {
+    # create an archive file for the previous copy of the db
+    # before the server starts up
+    mkdir -p "${HEKETI_DB_ARCHIVE_PATH}"
+
+    # only archive the db if the db content is valid
+    if ! $HEKETI_PATH/heketi db export --dbfile "${HEKETI_PATH}/heketi.db" --jsonfile - >/dev/null ; then
```

Comment:
Heketi path is /var/lib/heketi  and might not have heketi binary.

Reply:
I am also not sure if jsonfile takes "-". Need to test it out.

Reply:
Tested the jsonfile part, it creates a file with name "-".

Reply:
Weird. I thought I added that... Yep:
```
func DbDump(jsonfile string, dbfile string) error {

»»»»var fp *os.File
»»»»if jsonfile == "-" {
»»»»»»»»fp = os.Stdout
»»»»} else {
```
Are you testing with something other than master?

Reply:
Fixed the binary path. That was debugging stuff I accidentally left in.

Reply:
Thanks! I was using a heketi binary from a old released branch.